wolfcrypt/src/ecc.c

```diff
@@ -41,29 +41,51 @@ Possible ECC enable options:
  * HAVE_COMP_KEY:       Enables compressed key                  default: off
  * WOLFSSL_VALIDATE_ECC_IMPORT: Validate ECC key on import      default: off
  * WOLFSSL_CUSTOM_CURVES: Allow non-standard curves.            default: off
- *                         Includes the curve "a" variable in calculation
+ *                        Includes the curve "a" variable in calculation
+ * ECC_DUMP_OID:        Enables dump of OID encoding and sum   default: off
  */
 
 /*
-ECC Curves:
+ECC Curve Types:
+ * NO_ECC_SECP          Disables SECP curves                    default: on
```

Comment:
Yeah its not defined by default, which allows SECP to be used. I'll update that lingo. Thx